h/events.py

```diff
@@ -20,6 +20,7 @@ class AnnotationTransformEvent(object):
     annotation just before it is indexed or in other use-cases.
     """
 
-    def __init__(self, request, annotation_dict):
+    def __init__(self, request, annotation, annotation_dict):
         self.request = request
+        self.annotation = annotation
         self.annotation_dict = annotation_dict
```

Comment:
This looks a bit odd now. It's not really your fault and I completely understand why this change is as it is, but it definitely seems wrong to be passing an annotation and a serialisation of the same annotation into this event.